pages/documents/privacy/2021.tsx

```diff
@@ -0,0 +1,412 @@
+            </p>
+
+            <h3>Data Processing by Jacobs University</h3>
+            <p>
+                With your consent, we forward the data you have supplied to set up your user account to Jacobs University Bremen, with the exception of your payment data.
+                Jacobs University Bremen uses your data within the scope of the consent you have granted
+            </p>
+
+            <ul>
+                <li>to contact you on their own for networking purposes; and</li>
+                <li>and to inform you by e-mail or post about events, surveys, fundraising campaigns and other subjects relating to Jacobs University Bremen.</li>
+            </ul>
+
+            <p>
+                Only the following departments have access to your data: Business Solutions, Student Marketing & Recruitment, Academic Services, Academic Operations, Corporate Services, Campus Operations and Corporate Staff.
+                You are explicitly informed that your data is not intended for use by the Student Financial Services department.
+                Neither we nor Jacobs University Bremen will disclose your data to any other third parties.
+            </p>
+
+
+            <h3>Data Processing by Stripe Inc.</h3>
+            <p>
+                To enable processing of membership fess, we transfer a very limited set of information to our payment provider Stripe, Inc. 
+                This information is used to process membership fees and to issue donation receipts.
+            </p>
+
+
+
+
+            <h4>Data Transferred</h4>
+            <p>
+                The exact information transmitted can vary on a case-by-case basis, however we transfer at most the following information about you:
+            </p>
+
+            <ul>
+                <li>your full name, so that Stripe can identify who you are;</li>
+                <li>your existing email address, so that Stripe can send you a receipt and contact you in case of problems;</li>
+                <li>your selected membership tier, so that Stripe knows which amount to charge you for;</li>
+                <li>the date and time of your membership, so that Stripe knows when to bill you;</li>
+                <li>your selected payment method and payment details, so that Stripe knows how to charge you. This data includes</li>
+                <li>
+                    <ul>
+                        <li>your credit card details, i.e. your credit card number, expiration date and optionally your CVV (depending on the card type); or</li>
+                        <li>your SEPA bank account details, i.e. your bank account number (IBAN) and the owner of your Bank account.</li>
+                    </ul>
+                </li>
+            </ul>
+
+            <p>
+                The payment details are submitted directly from your browser to Stripe Inc. and the payment details are never stored by Jacobs University Alumni.
```

Comment:
"payment details are never stored by us".

Reply:
That is still true. We only store the payment history, not the payment details.

Reply:
Yes, I meant "Jacobs University Alumni" -> "us".